improve the hyperdrive getting started doc (#16553)
cloudflare/cloudflare-docs: src/content/docs/hyperdrive/get-started.mdx

Create a new project named `hyperdrive-tutorial` by running:

> {/* TODO: create a hyperdrive getting started template and ask the user to use that one */}

Author: ```suggestion
```

this can be a bit controversial and it's not really super needed here, let's just remove it for now

:::note

Note that the `wrangler.toml` file contains the following option:
```toml title="wrangler.toml"
compatibility_flags = [ "nodejs_compat" ]
```

> this enables the Node.js compatibility mode which is genrally required for database drivers to function.

Reviewer: ```suggestion
This enables the Node.js compatibility mode which is genrally required for database drivers to function.
```

Reviewer 2: Just say `required` - "generally" is ambiguous.

```suggestion
this enables the Node.js compatibility mode which is required for database drivers, including Postgres.js.
```

To create a Hyperdrive connection, run the `wrangler` command, replacing the placeholder values passed to the `--connection-string` flag with the values of your existing database:

```sh

> npx wrangler hyperdrive create YOUR_CONFIG_NAME --connection-string="postgres://user:password@HOSTNAME_OR_IP_ADDRESS:PORT/database_name"

Reviewer: ```suggestion
npx wrangler hyperdrive create <YOUR_CONFIG_NAME> --connection-string="postgres://user:password@HOSTNAME_OR_IP_ADDRESS:PORT/database_name"
```

In brackets "<>" is usually our model to indicate this must be replaced by the user

```

If successful, the command will output your new Hyperdrive configuration:

```json
{
	"id": "<example id: 57b7076f58be42419276f058a8968187>",

> 	"name": "YOUR_CONFIG_NAME",

Reviewer: ```suggestion
	"name": "<YOUR_CONFIG_NAME>",
```

	"origin": {

> 		"host": "YOUR_DATABASE_HOST",

Reviewer: ```suggestion
		"host": "<YOUR_DATABASE_HOST>",
```

To bind your Hyperdrive configuration to your Worker, add the following to the end of your `wrangler.toml` file:

```toml

> compatibility_flags = [ "nodejs_compat" ] # required for database drivers to function

Author: this has been covered earlier in the guide

			// Return result rows as JSON
			return Response.json(results);
		} catch (e) {
			console.error(e);
			return Response.json({ error: e instanceof Error ? e.message : e }, { status: 500 });
		}
	},
} satisfies ExportedHandler<Env>;
```

> The code above, upon receiving a request:

Reviewer: ```suggestion
Upon receiving a request, the code above does the following:
```

			// Return result rows as JSON
			return Response.json(results);
		} catch (e) {
			console.log(e);

> 			return Response.json({ error: e.message }, { status: 500 });

Author: this presents a TypeScript error since `e` is `unknown`
